src/Infrastructure.EntityFramework/AdminConsole/Repositories/Queries/OrganizationUserUserViewQuery.cs

```diff
@@ -28,9 +28,9 @@ from su in su_g.DefaultIfEmpty()
             SsoExternalId = x.su.ExternalId,
             Permissions = x.ou.Permissions,
             ResetPasswordKey = x.ou.ResetPasswordKey,
-            UsesKeyConnector = x.u != null && x.u.UsesKeyConnector,
+            UsesKeyConnector = x.u.UsesKeyConnector,
```

Comment:
This is causing the current failure in the CI test, because the [left join](https://github.com/bitwarden/server/pull/4896/files#r1823392614) on the `User` table can result in `UsesKeyConnector` being `NULL` - so it can't be assigned to a non-nullable value type. I suggest you revert this line or convert the null in some other way (e.g. `x.u.UsesKeyConnector == true`).